### PR TITLE
changelog: Cloud output v2

### PR DESCRIPTION
**Pull request by codebien**
Added the changelog's entry for the new cloud output.

### CODE REVIEW

**Comment by codebien** [comment]
@olegbespalov I'm opening this right now to discuss what we should communicate. I added a primary draft. Do you think we should detail more stuff here?

**Review by olegbespalov — APPROVED** [review]
Looks good, I think if something left we could iterate in the main release notes PR :rocket:

File: release notes/v0.46.0.md [hunk 1]
```diff
@@ -82,9 +82,17 @@ if (__ITER === 0) {
...
 
-### `<big_feature_1>` `#pr`
+### Cloud Output v2 [#3117](https://github.com/grafana/k6/issues/3117)
 
-_what, why, and what this means for the user_
+After years of great service, we decided to refresh the k6 Cloud output introducing a more efficient end-to-end solution for ingesting the generated tests' metrics.
+
+The new output reduces the load generators' used resources for tests that produce many metrics. There is no significant difference in the user experience; it's expected to be the same. 
+
+The one thing worth highlighting is that the new output is strict about tags and drops tags if they are reserved. It's `test_run_id` and any tag that starts with two underscores, `__` (Prometheus convention).
```
Comment (codebien) [inline release notes/v0.46.0.md:91]:
```suggestion
The one thing worth highlighting is that the new output is strict about tags and drops tags if they are reserved:
 - `test_run_id` as it is reserved for internal k6 cloud operations
 - any tag with a key that starts with two underscores (`__`), that is marked by Prometheus convention as reserved.
```
@olegbespalov WDYT? Could be an improvement?

**Review by codebien — COMMENTED** [review]
@olegbespalov great job, thanks :heart:

File: release notes/v0.46.0.md (hunk 1 above)
Comment (olegbespalov) [reply release notes/v0.46.0.md:91]: It's fine too me :+1:

**Pull request merged**